README.md

```diff
@@ -476,6 +476,44 @@ twilioVerify.processURL(snaUrl) { result in
 }
 ```
 
+## Privacy Manifest
+
+This document serves as the Privacy Manifest for the Twilio Verify SNA SDK. It outlines the privacy practices implemented in this SDK, providing a comprehensive understanding of how we handle data and respect user privacy.
+
+### Purpose
+
+The primary purpose of this Privacy Manifest is to facilitate developers and organizations in providing Apple with detailed information about the privacy practices employed within this SDK.
+
+### Usage
+
+To use this Privacy Manifest, simply refer to the relevant sections when you need to provide information to Apple or any other interested parties about the privacy practices used in this SDK.
+
+```xml
```

Comment:
Should we add a link to the file instead of adding the content here? I was thinking that changing the content should be applied to both instead of only having one source of truth

Reply:
Yeah I think it makes sense, I added a link instead of the content here. Thank you 👍